### ISSUE TITLE
what libraries are needed?

### ISSUE THREAD

**Issue by DarkStalke5r** [FredyJabe/aeyama#157]
### Mod Build

Idk

### Game Build

Releace build 146

### Issue Description

I installed the game not through github or liscord, but through the built-in mindusrty function, but the game does not start, are any libraries or additional modifications needed?

### Steps to Reproduce

1.install
2.run the game with the active mod (only "new world")

### Screenshots

![Screenshot_20241127-203455_Mindustry](https://github.com/user-attachments/assets/3c28a31f-2965-462e-bd92-f52bd29df2fa)

**Comment by SunkyMP3G** [comment]
It seems like you forgot to turn on the mod (the arrow button next to the trash can button)

**Comment by DarkStalke5r** [comment]
No, they say it was automatically disabled due to several game crashes.
P.s. I'm not a nozzle, I'm inattentive
